tests/stress/memory_limit.py

```diff
@@ -246,7 +257,7 @@ def execution_handler() -> None:
 
     with multiprocessing.Pool(processes=worker_count) as p:
         for worker in p.map(execute_function, workers):
-            log.info(f"Worker {worker.type} finished!")
+            print(f"Worker {worker.type} finished!")
```

Comment:
Why is `log.info` replaced with `print`?

Reply:
This was for my debugging, will remove it